rust/src/smb/detect.rs

```diff
@@ -201,3 +201,24 @@ pub extern "C" fn rs_smb_tx_get_dce_iface(state: &mut SMBState,
     }
     return 0;
 }
+
+#[no_mangle]
+pub unsafe extern "C" fn rs_smb_tx_get_filename(tx: &mut SMBTransaction,
+                                                buffer: *mut *const u8,
+                                                buffer_len: *mut u32)
+                                                -> u8
+{
+    match tx.type_data {
```

Comment:
`if let Some(SMBTransactionTypeData::CREATE(ref x)) = tx.type_data {`

Reply:
Changed in https://github.com/OISF/suricata/pull/7011/commits/392a666e125aaf602676a1256179c1ebd777fff8